explorer/src/Pos/Explorer/ExtraContext.hs

```diff
@@ -37,8 +86,91 @@ instance Monad m => HasGenesisRedeemAddressInfo (ExtraContextT m) where
         extraCtx <- Ether.ask @ExtraContext
         pure $ ecAddressCoinPairs extraCtx
 
-makeExtraCtx :: HasConfiguration => ExtraContext
-makeExtraCtx =
-    let addressCoinPairs = utxoToAddressCoinPairs $ unGenesisUtxo genesisUtxo
-        redeemOnly = filter (isRedeemAddress . fst) addressCoinPairs
-    in ExtraContext $ V.fromList redeemOnly
+-------------------------------------------------------------------------------------
+-- Explorer mock mode
+--
+-- The simple data structure that encapsulates functions that use CSL. We want to "cut"
+-- them out of the picture in order to be able to mock them.
+-------------------------------------------------------------------------------------
+
+data ExplorerMockMode = ExplorerMockMode
```

Comment:
Nice idea to have this ^! How about to re-name it to `ExplorerMockableMode` (note `able`)? It's a tiny change of the name, but with this we can say: Hey, we are using this guy not just in production, but also we are `able` to mock it for tests if needed. With `ExplorerMockMode` (w/o `\able\`) it sounds like we have something to use for mocking only (for me at least).

Reply:
Makes sense. I will change it.